ovos_solver_openai_persona/chat.py

```
@@ -0,0 +1,156 @@
+            if messages[0].role == MessageRole.SYSTEM:
+                if self.allow_system:  # merge system prompts
+                    sysm = AgentMessage(role=MessageRole.SYSTEM,
+                                        content=self.system_prompt + "\n" + messages[0].content)
+                # replace system prompt
+                messages[0] = sysm
+            else:
+                messages.insert(0, sysm)
+        return messages
+
+    def continue_chat(self, messages: List[AgentMessage],
+                      session_id: str = "default",
```

Ruff (0.14.14)

[warning] Unused method argument: `session_id`

(ARG002)